docs/howto/labels.md

```diff
@@ -0,0 +1,116 @@
+# How to define labels
+
+**Note**
+The term label may designate several concepts. Please check the
+[`ml:label` definition](/definitions/label).
+
+This documentation page covers the following topics:
+
+- Defining a [`ml:label`](/definitions/label), including a [class](/definitions/class).
+- Defining a label following the general definition "a kind of tag to
+  idendify an object" or identifying tag.
+
+## Defining a `ml:label`
+
+To define a field as being a [`ml:label`](/definitions/label), one must just
+add `ml:label` to its `dataType`s, for example:
+
+```json
+{
+    "@type": "ml:RecordSet",
+    "name": "weather_report",
+    ...
+    "field": [
+        ...,
+        {
+            "@type": "ml:Field",
+            "name": "air_temperature_c",
+            "description": "The air temperature, taken in the shadow.",
+            "dataType": ["sc:Number", "ml:label"],
+            "source": "..."
+        },
+    ],
+    ...
+}
+```
+
+In the above example, the `air_temperature_c` field is declared as being a
+`ml:label`, on top of being a number. This means that the dataset has
+originally been collected to train models that could predict the air
+temperature in ºC, given other parameters. Note that this does not prevent
+someone from using that same dataset for other purposes as predicting air
+temperature: the `ml:label` `dataType` is indicative only.
+
+### Defining a class
+
+A class is the possible value a discrete `ml:label` may take. Discrete
+values in Croissant are expressed using [enumerations](/howto/enumerations).
+
+So to define a class, one can just define an enumeration and specify the type
+`ml:label` on the field. For example:
+
+```json
+{
+          "@type": "ml:Field",
+          "name": "supercategory",
+          "description": "The name of the supercategory.",
+          "dataType": [
+            "sc:Text",
+            "sc:name",
+            "sc:label"
+          ],
+          "isEnumeration": true,
+}
+```
+
+In the above example, `supercategory` is a discrete text label, and the various
+supercategories are all classes.
+
+## Identifying tag
```

Comment:
For me it is counter-intuitive that the "Label" and the "Identifying tag" are described in the `labels.md` file, while we only want to refer to the former as a "label". Would it make sense to move this part to a separate `.md` file?